source/tools/testConsistency/ClientApp/app.test.js

```diff
@@ -0,0 +1,115 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. 
+// Licensed under the MIT License. 
+// To run these tests, first ensure that the flask app for nodeJs is running at 127.0.0.1 port 5000
+// and the flask app for .NET WPF is running at 127.0.0.1 port 4000. 
+const request = require("supertest");
+const app = require("./app");
+const fs = require("fs");
+
+test("The GET request to the index page should return a response code 200", function(done) {
+     request(app)
+      .get("/")
+      .then(function(response) {
+        expect(response.statusCode).toBe(200);
+        done();
+      });
+});
+
+test("Sending no card data should still return a response code 200", function(done) {
```

Comment:
>Sending no card data should still return a response code 200", [](start = 6, length = 62)

Nit nit: I think you can simply describe a very high level expecatation of success here (rather than going into 200 (which is implied here when talking about REST and successfult HTTP Status codes)-- So i am more interested here in the actual success scenario you are testing for - eg NonTemplatedCardsAreSupported or something like that as teh test name and the description which can explain "static cards without any template parameter specified should be accepted by the API" or something like that.